README.md

```diff
@@ -126,13 +134,14 @@ potentially, overlap the zones below.
 
 ![Drag item edit](/doc/img/edit-view-items.png)
 
-In the final step, you define the background and text color for drag
-items, as well as the drag items themselves. A drag item can contain
-either text or an image. You can define custom success and error
-feedback for each item. The feedback text is displayed in a popup
-after the learner drops the item on a zone - the success feedback is
-shown if the item is dropped on a correct zone, while the error
-feedback is shown when dropping the item on an incorrect drop zone.
+In the final step, you define the background and text color for drag items, as
+well as the drag items themselves. A drag item can contain either text or an
+image. You can define custom success and error feedback for each item. In
+standard mode, the feedback text is displayed in a popup after the learner drops
+the item on a zone - the success feedback is shown if the item is dropped on a
```

Comment:
I'm not seeing any success or failure feedback (except whether or not the item moves back to the bank) in standard mode on this problem:

http://studio-dndv2-sandbox8.opencraft.hosting/container/block-v1:Opencraft+DNDv2+v2.1+type@vertical+block@28ae4b11831b42949d8815c86671424b#

Aren't there supposed to be tooltips that show up?